server/src/cm_cmd_exec.cpp

```diff
@@ -203,9 +206,14 @@ cmd_spacedb (const char *dbname, T_CUBRID_MODE mode)
   int argc = 0;
   cubrid_err_file[0] = '\0';
 
-  find_and_parse_cub_admin_version (major_version, minor_version);
+  if (cubrid_version_minor <= 0)
```

Comment:
I think it is necessary to find and parse version information only when cubrid_version_minor is -1.
if (cubrid_version_minor == -1) or if (cubrid_version_minor <0)

Reply:
Thank you for your comment!
It seems my mistake, what I intended was:
if (cubrid_version_major <= 0)